Bitbucket.Authentication/Src/Authority.cs

```diff
@@ -69,7 +69,7 @@ public Authority(RuntimeContext context, string restRootUrl = null)
                 throw new ArgumentNullException(nameof(targetUri));
             if (credentials is null)
                 throw new ArgumentNullException(nameof(credentials));
-            if (resultType != AuthenticationResultType.Failure && resultType != AuthenticationResultType.Success && resultType != AuthenticationResultType.TwoFactor)
+            if (resultType < AuthenticationResultType.None || resultType > AuthenticationResultType.TwoFactor)
```

Comment:
Here's the important logic change. This fixes the issue.

Reply:
yep, I was just doing the same :) 

Thanks for expediting this. I will continue with my PR to add additional tests around this area to try and prevent issues in the future

Reply:
While this works, so does removing that check completely. Is it intended to provide some kind of future proofing against other result types?

Reply:
(I'm probably being a bit dim, not sure is there is an 'approve' button I'm supposed to click, can't see one, but I 'approve' 👍  )

Reply:
FWIW see https://github.com/Microsoft/Git-Credential-Manager-for-Windows/issues/651 as well

Reply:
@mminns there should be a "Start a Review" button, but I'll assume GitHub borked permissions and it is not appearing - so your ":+1:" is more than good enough for me. 😄

> I will continue with my PR to add additional tests around this area to try and prevent issues in the future

Thanks! Very much appreciated.